### PR TITLE
chore: bump Operator image version when issuing a release with new appVersion

### PR DESCRIPTION
**Pull request by Sig00rd**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,13 +38,14 @@ bump-version: sembump ## Bump the version in the version file. Set BUMP to [ pat
 	@echo "Updating version from $(VERSION) to $(NEW_VERSION) in README.md"
 	perl -i -pe 's/$(VERSION)/$(NEW_VERSION)/g' README.md
 
-.PHONE: change-chart-version
```
Comment (KorusMateusz) [inline Makefile:41]: XD

File: Makefile [hunk 2]
```diff
@@ -38,13 +38,14 @@ bump-version: sembump ## Bump the version in the version file. Set BUMP to [ pat
...
+.PHONY: change-chart-version
 change-chart-version: bump-version
 	@echo "+ $@"
 	$(eval VERSION=$(shell cat VERSION.txt))
 	sed -i "/version:/c\version: $(VERSION)" charts/carthago-op-jenkins/Chart.yaml
 	@if [ $(APP_VERSION) != $(OLD_APP_VERSION) ] ; then \
 		sed -i "/appVersion:/c\appVersion: \"$(APP_VERSION)\"" charts/carthago-op-jenkins/Chart.yaml ;\
+		sed -i "s/$(OLD_APP_VERSION)/$(APP_VERSION)/" charts/carthago-op-jenkins/values.yaml ;\
```
Comment (KorusMateusz) [inline Makefile:48]: Czemu ten sed jest inny niż ten wyżej?

**Review by KorusMateusz — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (prryb) [reply Makefile:41]: :spurdo_phone:

File: Makefile (hunk 2 above)
Comment (prryb) [reply Makefile:48]: If something else happens to use the same version number, we'll replace those as well.

**Review by prryb — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (Sig00rd) [reply Makefile]: fix'd!
@KorusMateusz /c\ replaces the whole line which is OK with `appVersion` and chart `version` but `image` is indented so it won't do

**Pull request merged**